tests/test-sessions/test-infinispan-sessions/pom.xml

```diff
@@ -11,7 +11,12 @@
   <url>http://www.eclipse.org/jetty</url>
   <properties>
     <bundle-symbolic-name>${project.groupId}.sessions.infinispan</bundle-symbolic-name>
-    <hotrod.host>127.0.0.1</hotrod.host>
+    <!-- if changing this version please update default in RemoteInfinispanTestSupport you will get thanks from Eclipse IDE users -->
+    <!-- 10.1.7.Final -->
+    <!--could be ${infinispan.version} but maybe we want to test different version of server vs client -->
+    <infinispan.docker.image.version>9.4.8.Final</infinispan.docker.image.version>
+    <!-- from 10.xx it has changed to jboss/infinispan -->
+    <infinispan.docker.image.name>jboss/infinispan-server</infinispan.docker.image.name>
```

Comment:
So the bit that is missing here (AFAICT) is the configuration of the cache on the remote infinispan. Currently, I (manually) configure and start a remote infinispan instance with the config file that I've attached here (note it should be called remote-session-test.xml, but github doesn't allow attachment of .xml files).
[remote-session-test.txt](https://github.com/eclipse/jetty.project/files/4608500/remote-session-test.txt)

Reply:
dohhh what a complicated configuration!! :) 
 what version are you using locally?
there can be some network issue with docker and how infinispan cluster mode works.
As it create on the fly some new endpoint and inform clients to use some ip/port (not sure to get this working here...)
Anyway I can try.

Reply:
I'm locally using 9.4.8.Final.  I just copied one of the existing configurations in the $INFINISPAN_HOME/standalone/configuration directory, renamed it, and added a cache called "remote-session-test".  I'm not sure how to strip it down to keep only hotrod protocol and the remote-session-test cache.

Reply:
I finally use your file but please note it was already working with the standalone mode :) 
But now it starts a cluster and we use it (please note the hotrod in is configured in BASIC mode to only use the pre defined host:port of the cluster  as we cannot use the dynamically created cluster node as we don't know in advance ports used so we cannot start docker with port mapping we don't know :)

Reply:
@olamy  I think there's some confusion here.  I merely copied the standalone.xml file to remote-session-test.xml , so if you're saying that standalone.xml from the distro is the default one that infinispan starts, then remote-session-test.xml should be _no more_ complicated, just with the addition of a cache named "remote-session-test".  Secondly, if you're saying that the standalone.xml file can be used _as is_ from the distro, and you're _absolutely sure_ it's using hotrod as the communication protocol and not defaulting to something else, then we can of course use it.

Reply:
yup I did some diff between the provided standalone.xml and remote-session-test.xml and they are very similar. Regarding hotrod See my comment here https://github.com/eclipse/jetty.project/pull/4862#discussion_r424344159

Reply:
I figure out we could change as well the test `RemoteQueryManagerTest` from `infinispan-remote-query`